dadi/frontend/components/FieldImage/FieldImageRoutes.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const config = require(paths.config)
+const AWS = require('aws-sdk')
+
+const ImageFieldRoute = function (app) {
+
```

Comment:
minor: we've not been using blank lines at the beginning of function declarations or any other blocks.